sdk/src/main/java/io/radar/sdk/RadarLocationManager.kt

```diff
@@ -553,7 +552,7 @@ internal class RadarLocationManager(
         val options = Radar.getTrackingOptions()
         val foregroundService = RadarSettings.getForegroundService(context)
 
-        if (foregroundService != null && foregroundService.updatesOnly) {
+        if (options.foregroundServiceEnabled && foregroundService.updatesOnly) {
```

Comment:
Since we migrated `trackingOptions.foregroundService` to `trackingOptions.foregroundServiceEnabled` and `Radar.setForegroundServiceOptions()`, we should be checking `options.foregroundServiceEnabled` instead of `foregroundService != null` to determine whether a foreground service should be started. Looks like we forgot to change that logic here.